src/openforms/submissions/exports.py

```diff
@@ -80,7 +80,7 @@ def create_submission_export(queryset: SubmissionQuerySet) -> tablib.Dataset:
 
 
 def export_submissions(
-    queryset: SubmissionQuerySet, file_type: FileType
+    queryset: models.QuerySet[Submission], file_type: FileType
```

Comment:
Why not keep `SubmissionQuerySet`?

Reply:
type error in the email plugin because it can only infer that a queryset of `Submission` is passed. The way the queryset is consumed here is compatible with a broader type and fixes the type error. I could not for the life of me make the manager return the `SubmissionQuerySet` type, it only wants to produce `models.QuerySet[Submission, Submission]` and it baffles me because I did the setup the same way like the payments manager and that one seems to work properly :thinking: 

So it's a compromise and I hope in a year from now there will be better built-in support